provider/cmd/pulumi-resource-aws/bridge-metadata.json

```diff
@@ -0,0 +1,3 @@
2hTZXQ6UmVnZXhNYXRjaFNldCI6MCwiYXdzOndhZi9yZWdleFBhdHRlcm5TZXQ6UmVnZXhQYXR0ZXJuU2V0IjowLCJhd3M6d2FmL3J1bGU6UnVsZSI6MCwiYXdzOndhZi9ydWxlR3JvdXA6UnVsZUdyb3VwIjowLCJhd3M6d2FmL3NpemVDb25zdHJhaW50U2V0OlNpemVDb25zdHJhaW50U2V0IjowLCJhd3M6d2FmL3NxbEluamVjdGlvbk1hdGNoU2V0OlNxbEluamVjdGlvbk1hdGNoU2V0IjowLCJhd3M6d2FmL3dlYkFjbDpXZWJBY2wiOjAsImF3czp3YWYveHNzTWF0Y2hTZXQ6WHNzTWF0Y2hTZXQiOjAsImF3czp3YWZyZWdpb25hbC9ieXRlTWF0Y2hTZXQ6Qnl0ZU1hdGNoU2V0IjowLCJhd3M6d2FmcmVnaW9uYWwvZ2VvTWF0Y2hTZXQ6R2VvTWF0Y2hTZXQiOjAsImF3czp3YWZyZWdpb25hbC9pcFNldDpJcFNldCI6MCwiYXdzOndhZnJlZ2lvbmFsL3JhdGVCYXNlZFJ1bGU6UmF0ZUJhc2VkUnVsZSI6MCwiYXdzOndhZnJlZ2lvbmFsL3JlZ2V4TWF0Y2hTZXQ6UmVnZXhNYXRjaFNldCI6MCwiYXdzOndhZnJlZ2lvbmFsL3JlZ2V4UGF0dGVyblNldDpSZWdleFBhdHRlcm5TZXQiOjAsImF3czp3YWZyZWdpb25hbC9ydWxlOlJ1bGUiOjAsImF3czp3YWZyZWdpb25hbC9ydWxlR3JvdXA6UnVsZUdyb3VwIjowLCJhd3M6d2FmcmVnaW9uYWwvc2l6ZUNvbnN0cmFpbnRTZXQ6U2l6ZUNvbnN0cmFpbnRTZXQiOjAsImF3czp3YWZyZWdpb25hbC9zcWxJbmplY3Rpb25NYXRjaFNldDpTcWxJbmplY3Rpb25NYXRjaFNldCI6MCwiYXdzOndhZnJlZ2lvbmFsL3dlYkFjbDpXZWJBY2wiOjAsImF3czp3YWZyZWdpb25hbC93ZWJBY2xBc3NvY2lhdGlvbjpXZWJBY2xBc3NvY2lhdGlvbiI6MCwiYXdzOndhZnJlZ2lvbmFsL3hzc01hdGNoU2V0Olhzc01hdGNoU2V0IjowLCJhd3M6d2FmdjIvaXBTZXQ6SXBTZXQiOjAsImF3czp3YWZ2Mi9yZWdleFBhdHRlcm5TZXQ6UmVnZXhQYXR0ZXJuU2V0IjowLCJhd3M6d2FmdjIvcnVsZUdyb3VwOlJ1bGVHcm91cCI6MCwiYXdzOndhZnYyL3dlYkFjbDpXZWJBY2wiOjAsImF3czp3YWZ2Mi93ZWJBY2xBc3NvY2lhdGlvbjpXZWJBY2xBc3NvY2lhdGlvbiI6MCwiYXdzOndhZnYyL3dlYkFjbExvZ2dpbmdDb25maWd1cmF0aW9uOldlYkFjbExvZ2dpbmdDb25maWd1cmF0aW9uIjowLCJhd3M6d29ya2xpbmsvZmxlZXQ6RmxlZXQiOjAsImF3czp3b3JrbGluay93ZWJzaXRlQ2VydGlmaWNhdGVBdXRob3JpdHlBc3NvY2lhdGlvbjpXZWJzaXRlQ2VydGlmaWNhdGVBdXRob3JpdHlBc3NvY2lhdGlvbiI6MCwiYXdzOndvcmtzcGFjZXMvZGlyZWN0b3J5OkRpcmVjdG9yeSI6MCwiYXdzOndvcmtzcGFjZXMvaXBHcm91cDpJcEdyb3VwIjowLCJhd3M6d29ya3NwYWNlcy93b3Jrc3BhY2U6V29ya3NwYWNlIjowLCJhd3M6eHJheS9lbmNyeXB0aW9uQ29uZmlnOkVuY3J5cHRpb25Db25maWciOjAsImF3czp4cmF5L2dyb3VwOkdyb3VwIjowLCJhd3M6eHJheS9zYW1wbGluZ1J1bGU6U2FtcGxpbmdSdWxlIjowfSwiZnVuY3Rpb25zIjp7fSwiY29uZmlnIjp7ImFjY2Vzc0tleSI6WzBdLCJhbGxvd2VkQWNjb3VudElkcyI6WzBdLCJhc3N1bWVSb2xlIjpbMF0sImFzc3VtZVJvbGVXaXRoV2ViSWRlbnRpdGllcyI6WzFdLCJhc3N1bWVSb2xlV2l0aFdlYklkZW50aXR5IjpbMF0sImFzc3VtZVJvbGVzIjpbMV0sImN1c3RvbUNhQnVuZGxlIjpbMF0sImRlZmF1bHRUYWdzIjpbMF0sImVjMk1ldGFkYXRhU2VydmljZUVuZHBvaW50IjpbMF0sImVjMk1ldGFkYXRhU2VydmljZUVuZHBvaW50TW9kZSI6WzBdLCJlbmRwb2ludHMiOlswXSwiZm9yYmlkZGVuQWNjb3VudElkcyI6WzBdLCJodHRwUHJveHkiOlswXSwiaWdub3JlVGFncyI6WzBdLCJpbnNlY3VyZSI6WzBdLCJtYXhSZXRyaWVzIjpbMF0sInByb2ZpbGUiOlswXSwicmVnaW9uIjpbMF0sInMzRm9yY2VQYXRoU3R5bGUiOlswXSwiczNVc2VQYXRoU3R5bGUiOlswXSwic2VjcmV0S2V5IjpbMF0sInNoYXJlZENvbmZpZ0ZpbGVzIjpbMF0sInNoYXJlZENyZWRlbnRpYWxzRmlsZSI6WzBdLCJzaGFyZWRDcmVkZW50aWFsc0ZpbGVzIjpbMF0sInNraXBDcmVkZW50aWFsc1ZhbGlkYXRpb24iOlswXSwic2tpcEdldEVjMlBsYXRmb3JtcyI6WzBdLCJza2lwTWV0YWRhdGFBcGlDaGVjayI6WzBdLCJza2lwUmVnaW9uVmFsaWRhdGlvbiI6WzBdLCJza2lwUmVxdWVzdGluZ0FjY291bnRJZCI6WzBdLCJzdHNSZWdpb24iOlswXSwidG9rZW4iOlswXSwidXNlRHVhbHN0YWNrRW5kcG9pbnQiOlswXSwidXNlRmlwc0VuZHBvaW50IjpbMF19fQ=="
```

Comment:
Erm, any reason this is opaque?

Reply:
Because the go json serialization layer encodes `[]byte` as base64. I'll fix that. It's a problem with the metadata system, not the muxer.